src/components/utils/getLazyLoadConfig.js

```diff
@@ -0,0 +1,16 @@
+export default function getLazyLoadConfig(config, id) {
+  if (!config?.slots) {
+    return null;
+  }
+  const slotConfig = config.slots.find((slot) => slot.id === id);
+  if (!slotConfig) {
+    return null;
+  }
+  if (
+    slotConfig.enableLazyLoad !== undefined &&
+    slotConfig.enableLazyLoad !== null
```

Comment:
nitpick but, `!= null` is the same thing as checking not equals null and undefined seperately.

Reply:
True, but the ESLint rule `eqeqeq` will complain if I use that. I agree with ESLint that it's better to always avoid JavaScript's type coercion to reduce mental overhead and risk of bugs.